### ISSUE TITLE
Name is not defined for the Combo boxes present on the page in Windows.

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#29316]
### Describe the bug

**Title:** Name is not defined for the Combo boxes present on the page in Windows.

**Pre-Requisite:**
Press F12 key to open developer tool.

**Test Environment:** 
OS: Windows 11 Version 24H2 OS build 26100.1742
Browser: Microsoft New Edge Version129.0.2792.52(Official build) (64-bit)
Tool: Developer Tool

**Actual Result:** 
Name is not defined for the 'Size' combo box present on the page in Windows. Screen reader is announcing as '16 combo box collapsed has pop up'.

Refer Attachment: 

https://github.com/user-attachments/assets/f963edd6-8843-4237-aaa5-c628cf087399

![Image](https://github.com/user-attachments/assets/73efa55a-99b4-4790-adec-23db70436496)

**Expected Result:** 
Name should be defined for the 'Size' combo box present on the page in Windows. Screen reader should announce as '16 size combo box collapsed has pop up'.

### Reproduction link

React Web Component Wrappers / Fluent Wrappers / Avatar - Docs ⋅ Storybook (ambitious-cliff-0c8148010.2.azurestaticapps.net) 

### Reproduction steps

**Repro Steps:**
Open URL: [React Web Component Wrappers / Fluent Wrappers / Avatar - Docs ⋅ Storybook (ambitious-cliff-0c8148010.2.azurestaticapps.net)](https://ambitious-cliff-0c8148010.2.azurestaticapps.net/?path=/docs/reactwrappers_react-web-component-wrappers-fluent-wrappers-avatar--docs) in latest Edge browser.
Story book page will open.
Press tab key from top of the page to reach the 'Size' combo box.
Press F12 to open developer tool.
Verify in developer tool whether name is defined for the 'Size' combo box or not.

### System

OS: Windows 11 Version 24H2 OS build 26100.1742
Browser: Microsoft New Edge Version129.0.2792.52(Official build) (64-bit)

**Comment by Sidnioulz** [comment]
Tracked in #31436, thanks for reporting!
